bedrock/base/templates/includes/global-nav.html

```diff
@@ -68,6 +68,33 @@ <h3 class="summary" data-id="firefox">
                 {{ _('Get the only browser built for people, not profit.') }}
               </p>
               <ul class="nav-menu-secondary-links">
+              {% if switch('firefox-57-release') %}
```

Comment:
@pmac this PR adds a switch in the base template, so will be queried on every page. Just checking this isn't going to cause some sort of perf disaster?

Reply:
Actually, it looks like we're already using switches in the base templates, so I'm guessing this is fine. Please disregard :)

Reply:
checking switches should be very fast now that they're not in the DB. Thanks for checking though! 💯